dadab/dadab.js

```diff
@@ -3,77 +3,101 @@ based on:
  - Simple Image Glitching Processing - datamoshing.com | 2016
  - a pixel sorting sketch from Jerome Martinez | 2017
  */
- 
+
 // output needs to be 1066 x 800
- 
+
+var capture;
+var gif;
 var img;
-var imgFileName = "vera";
-var fileType = "jpg";
+var imgFileName = 'vera';
+var fileType = 'jpg';
+
+var START_FRAME = 15;
+var END_FRAME = 115;
+
+const serverUrl = 'http://localhost:3000';
+
+const socketOptions = {
+  transports: ['websocket'],
+  'force new connection': true
+};
+const socket = io(serverUrl, socketOptions);
```

Comment:
All of these `const`s could be `var`s, but I copy-pasted out of some other code I had written.